common.js

```diff
@@ -49,7 +49,23 @@ function userIgnoreFilter (opts) {
     }
   }
 
+  var normalizedOut = opts.out ? path.resolve(opts.out) : null
+  var outIgnores = []
+  if (normalizedOut === null || normalizedOut === process.cwd()) {
+    ['darwin', 'linux', 'win32'].forEach(function (platform) {
+      ['ia32', 'x64'].forEach(function (arch) {
```

Comment:
I would move `supportedArchs` and `supportedPlatforms` in `index.js` to `common.js` and use those instead. Otherwise, that will make PRs like #107 and #223 more complex than they need to be.